doc/guides/coding_guidelines/index.rst

```diff
@@ -784,3 +787,26 @@ and the equivlent rules from other standards for reference.
       -  Where designated initializers are used to initialize an array object the size of the array shall be specified explicitly
       -  N/A
       -  `Rule 9.5 <https://gitlab.com/MISRA/MISRA-C/MISRA-C-2012/Example-Suite/-/blob/master/R_09_05.c>`_
+
+Additional rules
+================
+
+.. list-table:: Additional rules
```

Comment:
does it really need to be a table? this is very limiting and would make it difficult to add examples and additional context etc, maybe a definition list or some other type of list? the MISRA-C rules are in the table because they are references, this does not need to be the same for additional rules that we full maintain...

Reply:
I thought about this, yes. It doesn't really need to be a table, it's just that I thought consistency with the other guidelines looked nice. But I agree, and while I was making the table I was asking myself the same question. I will rewrite as a section-separated list.